### PR TITLE
Add unit test for Python version & code to set it

### LINKED ISSUE #33: Move `pyenv local 3.9.9` from CONTRIBUTING.md into `Makefile`
## Context

We nudge ourselves via `Makefile` to the correct expected Python version by adding awareness of `pyenv` to our `Makefile`.

`Makefile` in `awscli-login` has some relevant patterns.

### PR DESCRIPTION
**Pull request by ddriddle**
Closes #33

### CODE REVIEW

**Comment by ddriddle** [comment]
@edthedev if you want I can split this into two PR(s). One to test the version, the other to set the version in a DRY fashion.

**Review by edthedev — APPROVED** [review]

**Comment by edthedev** [comment]
@ddriddle I suspect that the current build error will clear if we set the OS back to Ubuntu LTS 20.04.

**Comment by edthedev** [comment]
> @edthedev if you want I can split this into two PR(s). One to test the version, the other to set the version in a DRY fashion.

I think it's fine as-is. It's all accomplishing the same underlying goal - being sure we're running on the Python we expect.

**Review by zdc217 — APPROVED** [review]

**Review by mpitcel — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -16,6 +16,13 @@ jobs:
     runs-on: ubuntu-latest
```
Comment (edthedev) [inline .github/workflows/deploy.yml:16]:
```suggestion
    runs-on: ubuntu-20.04
```

**Pull request merged**